Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CertificateStateHelper.kt

```diff
@@ -106,6 +106,7 @@ fun IncludeCertificateQrcodeCardBinding.bindValidityViews(
             qrTitle.isVisible = isPersonDetails
             qrSubtitle.isVisible = isPersonDetails
             startValidationCheckButton.isVisible = isPersonDetails
+            notificationBadge.isVisible = isPersonDetails
```

Comment:
notificationBadge.isVisible = `isPersonDetails && certificate.isNewlyRetrieved` ? 
Currently it is always visible in this screen

Reply:
Yes, sure 😅